lib/Extension/LanguageServerPsalm/LanguageServerPsalmExtension.php

```diff
@@ -85,13 +89,15 @@ public function configure(Resolver $schema): void
         ]);
         $schema->setTypes([
             self::PARAM_PSALM_BIN => 'string',
+            self::PARAM_PSALM_CONFIG => 'string',
```

Comment:
I think I'd remove this and keep the default as `null` to be consistent with other configurations where `null` => not declared.